lib/pages/media_player/lyrics_screen.dart

```diff
@@ -174,97 +174,99 @@ class __ActionButtonsState extends State<_ActionButtons> {
   bool _isInPlay = true;
   @override
   Widget build(BuildContext context) {
-    return Padding(
-      padding: const EdgeInsets.symmetric(horizontal: 20),
-      child: Column(
-        children: [
-          SliderTheme(
-            data: const SliderThemeData(
-              trackHeight: 2,
-              thumbShape: RoundSliderThumbShape(enabledThumbRadius: 6.0),
-              overlayShape: RoundSliderOverlayShape(overlayRadius: 0.0),
+    return SafeArea(
+      child: Padding(
+        padding: const EdgeInsets.symmetric(horizontal: 20),
+        child: Column(
+          children: [
+            SliderTheme(
+              data: const SliderThemeData(
+                trackHeight: 2,
+                thumbShape: RoundSliderThumbShape(enabledThumbRadius: 6.0),
+                overlayShape: RoundSliderOverlayShape(overlayRadius: 0.0),
+              ),
+              child: SizedBox(
+                width: MediaQuery.of(context).size.width,
+                child: Slider(
+                  min: 0,
+                  max: 100,
+                  activeColor: const Color.fromARGB(255, 230, 229, 229),
```

Comment:
Colors shd be used only in 2 formats
1) Standard Colors.red 
2) If its not standard, create it in constants.dart and import from there
Also only Colors.fromRGBO() must be used as our project convention

Please change this wheerever necessary